.travis.yml

```diff
@@ -208,7 +210,10 @@ script:
       ./configure $qmake_conf CXXFLAGS="$CXXFLAGS"
     else
       mkdir build && cd build
-      cmake $cmake_conf ../
+      # CMake from Kitware is installed in /usr/local/bin
```

Comment:
@sledgehammer999 that's the binary installer; the apt repo installs to `/usr/bin` as expected.